client/components/editor.vue

```diff
@@ -1,5 +1,29 @@
 <template>
   <div>
+    <v-dialog hide-overlay v-model="linkDialog" max-width="500px">
+      <v-card>
+        <v-card-text>
+          <v-form v-model="validLink" @submit.prevent="insertLink">
+            <v-text-field
+              autofocus
+              placeholder="http://"
+              v-model="link"
+              hint="Enter a link"
```

Comment:
Here you have hint "Enter a link", but below you have "Invalid URL." So which one should user enter? Link or URL? I think we should just say URL.